src/lib/core/utils/axisLabelUnit.ts

```diff
@@ -0,0 +1,15 @@
+import { Variable } from '../types/study';
```

Comment:
Nit: name this file `axis-label-unit`, to be consistent with the names of the other files in `utils`.

Reply:
Thank for you review 👍 I have changed the file name